### ISSUE TITLE
[v4] Second reload doesn't work

### ISSUE THREAD

**Issue by vojty** [gaearon/react-hot-loader#819]
### Description
I am having troubles with migration to webpack 3 and react-hot-loader v4.
The first reload works fine, but the second one fails with `Ignored an update to unaccepted module ./app/components/NewRoot.jsx -> ./app/Bridge.jsx`

app.jsx
```
import { Provider } from 'react-redux';
import Bridge from './Bridge';
...
    ReactDOM.render(
        <Provider store={store}>
            <Bridge history={history} />
        </Provider>,
        document.getElementById('app-analyze')
    );
```

bridge.jsx
```
import React from 'react';
import { hot } from 'react-hot-loader';
import { Route } from 'react-router-dom';
import { ConnectedRouter } from 'react-router-redux';

import NewRoot from './components/NewRoot';
import ConnectedIntlProvider from './utils/with_connected_intl'; // just React-intl IntlProvider wrapped with connect
const routes = <Route path="/" component={NewRoot} />;

const Bridge = ({ history }) => (
    <ConnectedIntlProvider>
        <ConnectedRouter history={history}>
            {routes}
        </ConnectedRouter>
    </ConnectedIntlProvider>
);

export default hot(module)(Bridge);
```

NewRoot.jsx
```
import React from 'react';
import { connect } from 'react-redux';
import { injectIntl } from 'react-intl';
import rootSelector from '../selectors/root_selector';

class NewRoot extends React.Component {
    render() {
        return (
            <div>
                ABC DEF
            </div>
        );
    }
}

const dispatchToProps = {
    test: () => {}
};

export default connect(rootSelector, dispatchToProps)(injectIntl(NewRoot));
```

.babelrc
```
{
  "presets": [
    [ "env", { "modules": false } ],
    "react",
    "stage-2"
  ],
  "env": {
    "test": {
      // Only for testing components (NODE_ENV = 'test') in NodeJS, which needs CJS to work
      "plugins": [ "transform-es2015-modules-commonjs" ]
    },
    "development": {
      "plugins": ["react-hot-loader/babel"]
    },
    "production": {
      "plugins": ["lodash"]
    }
  }
}
```

### Environment
```
...
    "react": "15.3.2",
    "react-dnd": "2.1.4",
    "react-dom": "15.3.2",
    "react-intl": "2.4.0",
    "react-redux": "5.0.6",
    "react-router": "4.2.0",
    "react-router-dom": "4.2.2",
    "react-router-redux": "5.0.0-alpha.9",
...
```

Any ideas what am I doing wrong?

**Comment by theKashey** [comment]
This is not related to React-hot-loader, only to webpacks HRM. If you will google "Ignored an update to unaccepted module " - you will found that a lot of people face it before, including the same case with react-hot-loader.
https://github.com/gaearon/react-hot-loader/issues/354, for example.
Unfortunately I did not saw any solution among all these issues, I could share with you.

Just double check you consume NewRoot, and how application works. Next you could try to run RHL v4 as v3 - they are using absolutely different approaches to "accept" the change.

**Comment by theKashey** [comment]
Should be bound to #820

**Comment by theKashey** [comment]
Yep, the same - redux causes stand-in to throw a warning, but logger is broken. As result - full page refresh.

**Comment by neoziro** [comment]
Fixed in 4.0.0-beta.18.

**Comment by vojty** [comment]
it works, thanks! :)
